apps/builder/app/builder/builder.tsx

```diff
@@ -432,8 +432,18 @@ export const Builder = ({
 
   const navigatorLayout = useNavigatorLayout();
   const dataLoadingState = useStore($dataLoadingState);
+  const [loadingState, setLoadingState] = useState($loadingState.get());
```

Comment:
better use useState(() => $loadingState.get()) to avoid one-time subscribe on each render